src/lib/StubLib.sol

```diff
@@ -153,10 +152,11 @@ library StubLib {
     {
         IExternalResolver resolverContract = $resolver.resolver;
 
-        if (address(resolverContract) != ZERO_ADDRESS) return;
-
-        if (resolverContract.resolveModuleRegistration({ sender: msg.sender, moduleAddress: moduleAddress, record: moduleRecord }) == false)
-        {
+        if (
+            address(resolverContract) != ZERO_ADDRESS
```

Comment:
@zeroknots some tests are throwing an error here because I changed the flow to check the resolver if its not `address(0)`. Before, if it was not `address(0)` it would not check the resolver. This change makes sense right?